services/minespace-web/src/components/Forms/projects/projectSummary/ProjectContacts.tsx

```diff
@@ -0,0 +1,263 @@
+import RenderField from "@mds/common/components/forms/RenderField";
+import RenderSelect from "@mds/common/components/forms/RenderSelect";
+import { CONTACTS_COUNTRY_OPTIONS } from "@mds/common";
+
+import { getDropdownProvinceOptions } from "@mds/common/redux/selectors/staticContentSelectors";
+
+const RenderContacts = (props) => {
+  const { fields, contacts } = props;
+  const dispatch = useDispatch();
+  const provinceOptions = useSelector(getDropdownProvinceOptions);
+  const handleClearProvince = (currentCountry, addressTypeCode, subDivisionCode, field) => {
+    // clear out the province if country has changed and it no longer matches
+    if (addressTypeCode !== currentCountry) {
+      const selectedProvince = currentCountry
+        ? provinceOptions.find((p) => p.value === subDivisionCode)
+        : {};
+
+      const contactFieldToBeChanged = `${field}.address.sub_division_code`;
+      if (addressTypeCode === "INT" || selectedProvince?.subType !== currentCountry) {
+        dispatch(change(FORM.ADD_EDIT_PROJECT_SUMMARY, contactFieldToBeChanged, null));
+      }
+    }
+  };
+
+  return (
+    <>
+      {fields.map((field, index) => {
+        const { address = {} } = contacts[index] ?? {};
+        const { address_type_code, sub_division_code } = address ?? {};
+        const isInternational = address_type_code === "INT";
+        const isPrimary = contacts[index].is_primary;
+        return (
+          // eslint-disable-next-line react/no-array-index-key
+          <div key={index}>
+            {index === 0 ? (
+              <>
+                <Typography.Title level={5}>Primary project contact</Typography.Title>
+                <Typography.Paragraph>
+                  Provide contact information for the person who has the main responsibility for
+                  coordinating this project.
+                </Typography.Paragraph>
+              </>
+            ) : (
+              <>
+                <Row gutter={16}>
+                  <Col span={10}>
+                    <Typography.Title level={5}>
+                      Additional project contact #{index}
+                    </Typography.Title>
+                  </Col>
+                  <Col span={12}>
+                    <Popconfirm
+                      placement="topLeft"
+                      title="Are you sure you want to remove this contact?"
+                      onConfirm={() => fields.remove(index)}
+                      okText="Remove"
+                      cancelText="Cancel"
+                    >
+                      <Button type="primary" size="small" ghost>
+                        <DeleteOutlined className="padding-sm--left icon-sm" />
+                      </Button>
+                    </Popconfirm>
+                  </Col>
+                </Row>
+              </>
+            )}
+            <Row gutter={16}>
+              <Col md={12} sm={24}>
+                <Field
+                  name={`${field}.first_name`}
```

Comment:
Other then the Typescript change, the other main changes in this file are adding:

- First_name field (line 87)
- Last_name field (line 96)
- Address fields. (line 157 - 218)
- Adding the handleClearProvince function (line 28)
- Removal of mapDispatchToProps  and mapStateToProps